packages/odyssey-react/src/components/Text/Text.tsx

```diff
@@ -114,6 +141,10 @@ let Text = forwardRef((props, ref) => {
     size = "base",
     wrap = "normal",
     lineHeight = "normal",
+    hoverColor,
+    focusColor,
+    disabledColor,
+    transition,
```

Comment:
I believe we'll want to assign defaults here and mention them in the above type JSDOC comments as well e.g. `@default normal`

Reply:
I'm hesitant to do that as text isn't interactive by default